src/test/java/org/jenkinsci/plugins/workflow/cps/DSLTest.java

```diff
@@ -420,6 +430,104 @@ public void namedSoleParamForStep() throws Exception {
+        CredentialsProvider.lookupStores(r.jenkins).iterator().next().addCredentials(Domain.global(), c);
+        String shellStep = Functions.isWindows()? "bat \"echo $PASSWORD\"\n" : "sh \"echo $PASSWORD\"\n";
+        p.setDefinition(new CpsFlowDefinition(""
+                + "node {\n"
+                + "withCredentials([usernamePassword(credentialsId: 'creds', usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {\n"
+                + shellStep
+                + "}\n"
+                + "}", true));
+        WorkflowRun run = r.assertBuildStatusSuccess(p.scheduleBuild2(0));
+        r.assertLogContains("Affected variable(s): [PASSWORD]", run);
+        InterpolatedSecretsAction reportAction = run.getAction(InterpolatedSecretsAction.class);
+        Assert.assertNotNull(reportAction);
+        Set<String> reportResults = reportAction.getResults();
+        MatcherAssert.assertThat(reportResults.size(), is(1));
+        MatcherAssert.assertThat(reportResults.iterator().next(), is("PASSWORD"));
+        LinearScanner scan = new LinearScanner();
+        FlowNode node = scan.findFirstMatch(run.getExecution().getCurrentHeads().get(0), new NodeStepTypePredicate(Functions.isWindows()? "bat" : "sh"));
+        ArgumentsAction argAction = node.getPersistentAction(ArgumentsAction.class);
+        Assert.assertFalse(argAction.isUnmodifiedArguments());
+        MatcherAssert.assertThat(argAction.getArguments().values().iterator().next(), instanceOf(ArgumentsAction.NotStoredReason.class));
+    }
+
+    @Test public void describableInterpolation() throws Exception {
+        final String credentialsId = "creds";
+        final String username = "bob";
+        final String password = "secr3t";
+        UsernamePasswordCredentialsImpl c = new UsernamePasswordCredentialsImpl(CredentialsScope.GLOBAL, credentialsId, "sample", username, password);
+        CredentialsProvider.lookupStores(r.jenkins).iterator().next().addCredentials(Domain.global(), c);
+        p.setDefinition(new CpsFlowDefinition(""
+                + "node {\n"
+                + "withCredentials([usernamePassword(credentialsId: 'creds', usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {\n"
+                + "archiveArtifacts(\"${PASSWORD}\")"
+                + "}\n"
+                + "}", true));
+        WorkflowRun run = r.assertBuildStatus(Result.FAILURE, p.scheduleBuild2(0));
+        r.assertLogContains("Affected variable(s): [PASSWORD]", run);
+        InterpolatedSecretsAction reportAction = run.getAction(InterpolatedSecretsAction.class);
+        Assert.assertNotNull(reportAction);
+        Set<String> reportResults = reportAction.getResults();
+        MatcherAssert.assertThat(reportResults.size(), is(1));
+        MatcherAssert.assertThat(reportResults.iterator().next(), is("PASSWORD"));
+        // TODO: Code below currently fails
```

Comment:
I would delete these commented-out lines and add them in the other PR, same for `describableNoMetaStep`.

Reply:
removed in 70c4c223dd4d08ca065fb9f8b0e41bb3ccc07c6d